tests/integration/mixer/telemetry/metrics/passthrough/passthrough_test.go

```diff
@@ -0,0 +1,117 @@
+//  Copyright 2020 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package blackhole
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/mixer"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/prometheus"
+	"istio.io/istio/pkg/test/framework/components/sleep"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	util "istio.io/istio/tests/integration/mixer"
+)
+
+var (
+	sleepNs    namespace.Instance
+	ist        istio.Instance
+	prom       prometheus.Instance
+)
+
+func TestPassthroughClusterMetric(t *testing.T) {
+	framework.
+		NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			sleepInst := sleep.DeployOrFail(t, ctx, sleep.Config{Namespace: sleepNs, Cfg: sleep.Sleep})
+			respCode, err := sleepInst.Curl("http://istio.io")
```

Comment:
we can avoid external dependencies by using Sidecar and still test this, see pilot/outboundpolicytest. In fact, maybe we should merge these tests?

I commented this without reading the rest of the PR, I'll be more helpful tomorrow when I have time to review

Reply:
Just use some prow link. If the test runs, it means prow is alive.

Reply:
See comment https://github.com/istio/istio/pull/22021#issuecomment-599726103